specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2021-01-28-preview/HybridCompute.json

```diff
@@ -0,0 +1,1205 @@
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      }
+    },
+    "ErrorDetail": {
+      "title": "Error details.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error's code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "A human readable error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "Indicates which property in the request is responsible for the error.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Additional error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "ErrorResponse": {
+      "title": "Error response.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "$ref": "#/definitions/ErrorDetail"
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Identity": {
+      "title": "Managed Identity.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The identity type.",
+          "type": "string"
+        },
+        "principalId": {
+          "readOnly": true,
+          "description": "The identity's principal id.",
+          "type": "string"
+        },
+        "tenantId": {
+          "readOnly": true,
+          "description": "The identity's tenant id.",
+          "type": "string"
+        }
+      }
+    },
+    "MachineExtension": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "allOf": [
+            {
+              "$ref": "#/definitions/MachineExtensionProperties"
+            }
+          ],
+          "description": "Describes Machine Extension Properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Describes a Machine Extension."
+    },
+    "MachineExtensionUpdate": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "allOf": [
+            {
+              "$ref": "#/definitions/MachineExtensionUpdateProperties"
+            }
+          ],
+          "description": "Describes Machine Extension Update Properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResource"
+        }
+      ],
+      "description": "Describes a Machine Extension Update."
+    },
+    "MachineExtensionProperties": {
+      "properties": {
+        "forceUpdateTag": {
```

Comment:
What are the possible values for this?  This feels like something where there are know values the user should pass.  This should be documented as a closed or open enum

Reply:
I'd like to defer any changes to Extensions for now. these have been in for a long time and several api versions, rather than being added or edited in this api version. I'm not clear why these were approved previously if they're not acceptable now, but i'd like to pass this over to a SME in the area on our team for a follow up PR, rather than blocking the changes this PR was intended for

Reply:
This is mainly a usability concern - how does the user know what values this might contain.  It is ok to defer for now